app/gui2/src/components/GraphEditor.vue

```diff
@@ -269,22 +271,23 @@ const mouseHandler = nodeBindings.mouseHandler(
         :node="node"
         :selected="selectedNodes.has(id)"
         :is-latest-selected="id === latestSelectedNode"
-        :floating-node-container-node="floatingNodeContainerNode"
-        @update:selected="setSelected(id, $event), $event && (latestSelectedNode = id)"
-        @replaceSelection="selectedNodes.clear(), selectedNodes.add(id), (latestSelectedNode = id)"
+        @update:selected="setSelected(id, $event), $event && updateLatestSelectedNode(id)"
+        @replaceSelection="
+          selectedNodes.clear(), selectedNodes.add(id), updateLatestSelectedNode(id)
+        "
```

Comment:
these are pretty funky... on one hand, i guess the right thing to do is to extract them to a function
on the other hand i don't think it's a bad thing to have the definitions closer to (or rather, exactly at) the place where they are used, *if* the definitions are simple enough